### PR TITLE
Pass logits instead of probs to CrossEntroy

### LINKED ISSUE #1204: Wrong cross_entropy_loss computation in the text_matching example
开发者你好，
我在参考你们的example代码的时候，发现text_matching的示例中，cross entropy loss的计算有点问题。
按照官网的介绍，paddle.nn.CrossEntropyLoss的输入应当是logits和label，但是示例中给的是probs。
我在ernie_matching 和sentence_transformers 两个示例中均发现了这个问题。
 
https://github.com/PaddlePaddle/PaddleNLP/blob/27917453b598dec9791f692b0a72395a19ba95fe/examples/text_matching/ernie_matching/model.py#L40

https://github.com/PaddlePaddle/PaddleNLP/blob/27917453b598dec9791f692b0a72395a19ba95fe/examples/text_matching/ernie_matching/train_pointwise.py#L166

### PR DESCRIPTION
**Pull request by tianxin1860**
### PR types

Bug Fixes
### PR changes

Models
### Description

1. pass logits instead of probs to CrossEntroy.
2. fix #1204

### CODE REVIEW

**Review by ZeyuChen — APPROVED** [review]
LGTM
